src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/SavePublishProfileRequest.cs

```diff
@@ -0,0 +1,54 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using System.Collections.Generic;
+using Microsoft.SqlServer.Dac;
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+//using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.DacFx.Contracts
+{
+    /// <summary>
+    /// Parameters for a DacFx save publish profile request.
+    /// </summary>
+    public class SaveProfileParams
```

Comment:
`SavePublishProfileParams` to be consistent with the request name and file name

Reply:
Done.